### DISCUSSION TITLE
Remove requirements for branch naming

### DISCUSSION THREAD

**Discussion by sebastian-peter** [ie3-institute/simona#903] [Ideas]
Currently, we have to structure branch names with name initials, issue number etc. (`<initials>/#<issue number>-<issue description>`).

This requirement was established way back when we were still working with internal gitea and jira for issues, thus issues and PRs could be linked somehow. Nowadays, this link is established with Github PRs (including assigned person), thus we could drop the requirement and simplify the workflow.

Not sure how to go about release branches, it's maybe best to keep the structure `rel/<description>`.

Please le me know if you agree or not.

**Comment by t-ober** [comment]
Would the new convention then be `<description>` ? 

I find the initials handy since they create a folder in the Intellij git oveview: 
<img width="466" alt="image" src="https://github.com/user-attachments/assets/e81bdcb1-5118-4497-a8e9-2fdab0acaee0">

Also they make it easier to track back the branch in the console tab suggestions:
<img width="873" alt="image" src="https://github.com/user-attachments/assets/4181b28e-cc9e-4072-abfe-47379e62d2ee">

And I use the number for the: 
`Resolves #XYZ` bit in the PR (although there might be a different way to do that easily)

**Reply by sebastian-peter** [reply]
Ok, if there's use for the additional information, we can just keep it like it is for now

**Reply by danielfeismann** [reply]
I agree to Thomas. The folder creation in Intellij is realy handy. Same to have another link between issue and pr not only via Github. So I would prefer to keep it like it is.

**Comment by sebastian-peter** [comment]
Since there's preferences to keep things like they are, I'm closing this discussion now
